avocado/utils/asset.py

```diff
@@ -65,7 +65,8 @@ def __init__(self, name, asset_hash, algorithm, locations, cache_dirs,
         :param name: the asset filename. url is also supported
         :param asset_hash: asset hash
         :param algorithm: hash algorithm
-        :param locations: list of locations where the asset can be fetched from
+        :param locations: list of locations/single location Url as String
+                          where the asset can be fetched from
```

Comment:
Shall we rephrase this? We don't mention types for the other arguments... we can also omit that here now. Something like:
```
:param locations: location(s) where the asset can be fetched from
```
Considering that we now support basically anything, that's accurate and consistent.

Reply:
yes sure that right